.github/workflows/build.yaml

```diff
@@ -41,7 +41,6 @@ jobs:
       if: ${{ always() }}
       run: yes | docker image prune
     - name: Push an image to DockerHub
-      run: echo Pushing "${{ github.event.inputs.image }}" ${{ github.event.inputs.version }}
       run: ./ci.sh push "${{ github.event.inputs.image }}" ${{ github.event.inputs.version }}
```

Comment:
This seems to be a change that was meant to be in the PR $42 to enhance the BUILD IMAGE FROM TEMPLATE workflow, but it didn't make it into that PR, so you're sneaking it into this PR. ;-)  That's fine, just checking to make sure I understand what's going on...

Reply:
hmmm, this one was just cleaning up a mistake I made somewhere but didn't realise at the time.